llvm.proj

```diff
@@ -139,6 +143,10 @@
     <_LLVMBuildArgs Condition="'$(BuildOS)' == 'OSX' and '$(TargetArchitecture)' == 'x64'" Include='-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13' />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildOS)' == 'Windows_NT'">
+    <_LLVMBuildArgs Include='-DCMAKE_SHARED_LINKER_FLAGS_INIT="$(_SharedLinkerFlags)"' />
```

Comment:
as Jan pointed out in the runtime PR we'll need to add this to the compiler too so either through CMAKE_C_FLAGS/CMAKE_CXX_FLAGS or CMAKE_C_FLAGS_INIT/CMAKE_CXX_FLAGS_INIT